app/components/e-panel.vue

```diff
@@ -0,0 +1,543 @@
+       */
+      color: {
+        type: String,
+        default: 'yellow',
+        validator(value) {
+          return [
+            'yellow',
+            'blue',
+            'gray'
+          ].includes(value);
+        },
+      },
+
+      /**
+       * Defines if the triangle shaped button shall be displayed or not
+       */
+      triangleButton: {
+        type: Boolean,
+        default: false
+      },
+
+      /**
+       * Defines the size of the triangle button
+       *
+       * Valid values: `small`, `big`
+       */
+      triangleButtonSize: {
+        type: String,
+        default: 'small',
+        validator(value) {
+          return [
+            'small',
+            'big'
+          ].includes(value);
+        },
+      },
+
+      /**
+       * Defines the plus color of the plus for the triangle button
+       *
+       * Valid values: `white`, `blue`
+       */
+      triangleButtonPlusColor: {
+        type: String,
+        default: 'white',
+        validator(value) {
+          return [
+            'white',
+            'blue'
+          ].includes(value);
+        },
+      },
+
+    },
+    // data() {
+    //   return {};
+    // },
+
+    computed: {
+      hasHeadline() {
+        return this.headline !== null && this.headline !== '';
+      }
+    },
+    // watch: {},
+
+    // beforeCreate() {},
+    // created() {},
+    // beforeMount() {},
+    // mounted() { },
+    // beforeUpdate() {},
+    // updated() {},
+    // activated() {},
+    // deactivated() {},
+    // beforeDestroy() {},
+    // destroyed() {},
+
+    methods: {
+      mouseOver(event) {
+        const plus = event.target.classList.contains('e-panel__plus') ? event.target : event.target.firstChild;
+        const button = plus.parentNode;
+
+        plus.classList.add('e-panel__plus--hover');
+        button.classList.add('e-panel__triangle-button--hover');
+      },
+      mouseOut(event) {
+        const plus = event.target.classList.contains('e-panel__plus') ? event.target : event.target.firstChild;
+        const button = plus.parentNode;
+
+        plus.classList.remove('e-panel__plus--hover');
+        button.classList.remove('e-panel__triangle-button--hover');
+      },
+    },
+    // render() {},
+  };
+</script>
+
+<style lang="scss">
+  .e-panel {
+    position: relative;
+    overflow: hidden;
+    background-color: $color-grayscale--1000;
+
+    &--contour-thin {
+      border-bottom: 1px solid;
+    }
+
+    &--contour-thick {
+      border-bottom: 2px solid;
+    }
+
+    &--color-yellow {
+      border-color: $color-primary--1;
+    }
+
+    &--color-blue {
+      border-color: $color-secondary--1;
+    }
+
+    &--color-gray {
+      border-color: $color-grayscale--600;
+    }
+
+    &__headline {
+      padding: $spacing--10;
+    }
+
+    &__headline-underline {
+      /* TODO - should we define line-height variables? */
+      @include font($font-size: $font-size--18, $line-height: 23px);
```

Comment:
Asked myself the same a few times, but I think it would not be of to much help. line-height often is not as consistent as font-size and can also be used for styling purposes (center vertically). I think we will leave this for now. By default the line-height is `1.25` which should be ok in most cases.

Reply:
- [ ] Remove variable assignment `$font-size/$line-height`. They are only needed if you skip a property when calling a method, which is not the case here. (`font(<font-size>, <line-height>, <font-weight>)`)

Reply:
Sorry, don't know what you are referring to here.